<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Dialog</class>
 <widget class="QDialog" name="Dialog">
  <property name="windowModality">
   <enum>Qt::NonModal</enum>
  </property>
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>270</width>
    <height>305</height>
   </rect>
  </property>
  <property name="baseSize">
   <size>
    <width>0</width>
    <height>255</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>ProxyCare</string>
  </property>
  <property name="windowIcon">
   <iconset resource="systray.qrc">
    <normaloff>:/images/logo-on.svg</normaloff>:/images/logo-on.svg</iconset>
  </property>
  <property name="sizeGripEnabled">
   <bool>false</bool>
  </property>
  <property name="modal">
   <bool>false</bool>
  </property>
  <widget class="QTabWidget" name="tabWidget">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>100</y>
     <width>251</width>
     <height>161</height>
    </rect>
   </property>
   <property name="tabPosition">
    <enum>QTabWidget::South</enum>
   </property>
   <property name="tabShape">
    <enum>QTabWidget::Rounded</enum>
   </property>
   <property name="currentIndex">
    <number>0</number>
   </property>
   <property name="elideMode">
    <enum>Qt::ElideNone</enum>
   </property>
   <property name="documentMode">
    <bool>false</bool>
   </property>
   <widget class="QWidget" name="tab1">
    <attribute name="icon">
     <iconset>
      <normalon>:/images/proxy-icon.png</normalon>
     </iconset>
    </attribute>
    <attribute name="title">
     <string/>
    </attribute>
    <attribute name="toolTip">
     <string>Configurar proxy.</string>
    </attribute>
    <widget class="QFrame" name="frame2">
     <property name="geometry">
      <rect>
       <x>3</x>
       <y>3</y>
       <width>241</width>
       <height>121</height>
      </rect>
     </property>
     <property name="frameShape">
      <enum>QFrame::Box</enum>
     </property>
     <property name="frameShadow">
      <enum>QFrame::Raised</enum>
     </property>
     <widget class="QLineEdit" name="proxyLineEdit">
      <property name="geometry">
       <rect>
        <x>43</x>
        <y>44</y>
        <width>120</width>
        <height>29</height>
       </rect>
      </property>
      <property name="toolTip">
       <string>Servidor proxy remoto.</string>
      </property>
     </widget>
     <widget class="QSpinBox" name="proxyPortSpinBox">
      <property name="geometry">
       <rect>
        <x>168</x>
        <y>45</y>
        <width>64</width>
        <height>27</height>
       </rect>
      </property>
      <property name="toolTip">
       <string>Puerto del servidor proxy remoto.</string>
      </property>
      <property name="minimum">
       <number>1</number>
      </property>
      <property name="maximum">
       <number>9999</number>
      </property>
      <property name="value">
       <number>8080</number>
      </property>
     </widget>
     <widget class="QLineEdit" name="domainLineEdit">
      <property name="geometry">
       <rect>
        <x>42</x>
        <y>7</y>
        <width>190</width>
        <height>29</height>
       </rect>
      </property>
      <property name="toolTip">
       <string>Dominio.</string>
      </property>
     </widget>
     <widget class="QLabel" name="domainLabel">
      <property name="geometry">
       <rect>
        <x>6</x>
        <y>6</y>
        <width>31</width>
        <height>31</height>
       </rect>
      </property>
      <property name="toolTip">
       <string>Dominio</string>
      </property>
      <property name="text">
       <string/>
      </property>
      <property name="pixmap">
       <pixmap resource="systray.qrc">:/images/domain-icon.png</pixmap>
      </property>
      <property name="scaledContents">
       <bool>true</bool>
      </property>
     </widget>
     <widget class="QLabel" name="proxyLabel">
      <property name="geometry">
       <rect>
        <x>6</x>
        <y>42</y>
        <width>33</width>
        <height>31</height>
       </rect>
      </property>
      <property name="toolTip">
       <string>Servidor proxy.</string>
      </property>
      <property name="text">
       <string/>
      </property>
      <property name="pixmap">
       <pixmap resource="systray.qrc">:/images/proxy-icon.png</pixmap>
      </property>
      <property name="scaledContents">
       <bool>true</bool>
      </property>
     </widget>
     <widget class="QLabel" name="portLabel">
      <property name="geometry">
       <rect>
        <x>5</x>
        <y>79</y>
        <width>33</width>
        <height>31</height>
       </rect>
      </property>
      <property name="toolTip">
       <string>Puerto del servidor proxy local.</string>
      </property>
      <property name="text">
       <string/>
      </property>
      <property name="pixmap">
       <pixmap resource="systray.qrc">:/images/port-icon.png</pixmap>
      </property>
      <property name="scaledContents">
       <bool>true</bool>
      </property>
     </widget>
     <widget class="QSpinBox" name="portSpinBox">
      <property name="geometry">
       <rect>
        <x>43</x>
        <y>81</y>
        <width>64</width>
        <height>27</height>
       </rect>
      </property>
      <property name="toolTip">
       <string>Puerto del servidor proxy local.</string>
      </property>
      <property name="minimum">
       <number>1024</number>
      </property>
      <property name="maximum">
       <number>9999</number>
      </property>
      <property name="value">
       <number>3128</number>
      </property>
     </widget>
    </widget>
   </widget>
   <widget class="QWidget" name="tab2">
    <attribute name="icon">
     <iconset>
      <normalon>:/images/cuote-icon.png</normalon>
     </iconset>
    </attribute>
    <attribute name="title">
     <string/>
    </attribute>
    <attribute name="toolTip">
     <string>Estado de la cuenta.</string>
    </attribute>
    <widget class="QFrame" name="frame3">
     <property name="geometry">
      <rect>
       <x>3</x>
       <y>3</y>
       <width>241</width>
       <height>121</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true"/>
     </property>
     <property name="frameShape">
      <enum>QFrame::Box</enum>
     </property>
     <property name="frameShadow">
      <enum>QFrame::Raised</enum>
     </property>
     <widget class="QProgressBar" name="progressBar">
      <property name="enabled">
       <bool>true</bool>
      </property>
      <property name="geometry">
       <rect>
        <x>42</x>
        <y>10</y>
        <width>190</width>
        <height>29</height>
       </rect>
      </property>
      <property name="autoFillBackground">
       <bool>false</bool>
      </property>
      <property name="styleSheet">
       <string notr="true"/>
      </property>
      <property name="maximum">
       <number>0</number>
      </property>
      <property name="value">
       <number>0</number>
      </property>
      <property name="textVisible">
       <bool>true</bool>
      </property>
      <property name="orientation">
       <enum>Qt::Horizontal</enum>
      </property>
      <property name="invertedAppearance">
       <bool>false</bool>
      </property>
      <property name="textDirection">
       <enum>QProgressBar::TopToBottom</enum>
      </property>
      <property name="format">
       <string>%v/%m</string>
      </property>
     </widget>
     <widget class="QLabel" name="lbl_1">
      <property name="geometry">
       <rect>
        <x>6</x>
        <y>8</y>
        <width>31</width>
        <height>31</height>
       </rect>
      </property>
      <property name="toolTip">
       <string>Estado de la cuenta.</string>
      </property>
      <property name="text">
       <string/>
      </property>
      <property name="pixmap">
       <pixmap resource="systray.qrc">:/images/cuote-icon.png</pixmap>
      </property>
      <property name="scaledContents">
       <bool>true</bool>
      </property>
     </widget>
     <widget class="QLabel" name="lbl_2">
      <property name="geometry">
       <rect>
        <x>6</x>
        <y>83</y>
        <width>31</width>
        <height>31</height>
       </rect>
      </property>
      <property name="toolTip">
       <string>Intervalo de actualización.</string>
      </property>
      <property name="text">
       <string/>
      </property>
      <property name="pixmap">
       <pixmap resource="systray.qrc">:/images/clock-icon.png</pixmap>
      </property>
      <property name="scaledContents">
       <bool>true</bool>
      </property>
     </widget>
     <widget class="QSpinBox" name="spin1">
      <property name="geometry">
       <rect>
        <x>42</x>
        <y>86</y>
        <width>71</width>
        <height>27</height>
       </rect>
      </property>
      <property name="toolTip">
       <string>Intervalo de actualización.</string>
      </property>
      <property name="suffix">
       <string> min</string>
      </property>
      <property name="prefix">
       <string/>
      </property>
      <property name="minimum">
       <number>1</number>
      </property>
      <property name="value">
       <number>1</number>
      </property>
     </widget>
     <widget class="QLabel" name="nav_level">
      <property name="geometry">
       <rect>
        <x>40</x>
        <y>40</y>
        <width>191</width>
        <height>20</height>
       </rect>
      </property>
      <property name="text">
       <string>Desconectado</string>
      </property>
      <property name="alignment">
       <set>Qt::AlignCenter</set>
      </property>
     </widget>
     <widget class="Line" name="line">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>68</y>
        <width>221</width>
        <height>16</height>
       </rect>
      </property>
      <property name="orientation">
       <enum>Qt::Horizontal</enum>
      </property>
     </widget>
     <widget class="QPushButton" name="cmdRefresh">
      <property name="enabled">
       <bool>false</bool>
      </property>
      <property name="geometry">
       <rect>
        <x>202</x>
        <y>83</y>
        <width>31</width>
        <height>31</height>
       </rect>
      </property>
      <property name="mouseTracking">
       <bool>true</bool>
      </property>
      <property name="focusPolicy">
       <enum>Qt::NoFocus</enum>
      </property>
      <property name="toolTip">
       <string>Actualizar.</string>
      </property>
      <property name="text">
       <string/>
      </property>
      <property name="icon">
       <iconset resource="systray.qrc">
        <normaloff>:/images/refresh-icon.png</normaloff>:/images/refresh-icon.png</iconset>
      </property>
      <property name="iconSize">
       <size>
        <width>32</width>
        <height>32</height>
       </size>
      </property>
     </widget>
    </widget>
   </widget>
   <widget class="QWidget" name="tab3">
    <attribute name="icon">
     <iconset>
      <normalon>:/images/options.png</normalon>
     </iconset>
    </attribute>
    <attribute name="title">
     <string/>
    </attribute>
    <attribute name="toolTip">
     <string>Configurar ProxyCare.</string>
    </attribute>
    <widget class="QFrame" name="frame3_2">
     <property name="geometry">
      <rect>
       <x>3</x>
       <y>3</y>
       <width>241</width>
       <height>121</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true"/>
     </property>
     <property name="frameShape">
      <enum>QFrame::Box</enum>
     </property>
     <property name="frameShadow">
      <enum>QFrame::Raised</enum>
     </property>
     <widget class="QCheckBox" name="checkBox1">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>10</y>
        <width>230</width>
        <height>31</height>
       </rect>
      </property>
      <property name="text">
       <string>Conectar automáticamente
al iniciar.</string>
      </property>
      <property name="checked">
       <bool>false</bool>
      </property>
     </widget>
     <widget class="QCheckBox" name="checkBox2">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>50</y>
        <width>230</width>
        <height>31</height>
       </rect>
      </property>
      <property name="text">
       <string>Ocultar al iniciar.</string>
      </property>
      <property name="checked">
       <bool>false</bool>
      </property>
     </widget>
    </widget>
   </widget>
  </widget>
  <widget class="QFrame" name="frame1">
   <property name="geometry">
    <rect>
     <x>15</x>
     <y>4</y>
     <width>241</width>
     <height>91</height>
    </rect>
   </property>
   <property name="frameShape">
    <enum>QFrame::NoFrame</enum>
   </property>
   <property name="frameShadow">
    <enum>QFrame::Raised</enum>
   </property>
   <widget class="QLabel" name="passLabel">
    <property name="geometry">
     <rect>
      <x>6</x>
      <y>39</y>
      <width>31</width>
      <height>31</height>
     </rect>
    </property>
    <property name="toolTip">
     <string>Contraseña.</string>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="pixmap">
     <pixmap resource="systray.qrc">:/images/passw-icon.png</pixmap>
    </property>
    <property name="scaledContents">
     <bool>true</bool>
    </property>
   </widget>
   <widget class="QLineEdit" name="userLineEdit">
    <property name="geometry">
     <rect>
      <x>42</x>
      <y>4</y>
      <width>190</width>
      <height>29</height>
     </rect>
    </property>
    <property name="focusPolicy">
     <enum>Qt::StrongFocus</enum>
    </property>
    <property name="toolTip">
     <string>Usuario.</string>
    </property>
   </widget>
   <widget class="QLabel" name="userLabel">
    <property name="geometry">
     <rect>
      <x>6</x>
      <y>2</y>
      <width>31</width>
      <height>31</height>
     </rect>
    </property>
    <property name="toolTip">
     <string>Usuario.</string>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="pixmap">
     <pixmap resource="systray.qrc">:/images/user.png</pixmap>
    </property>
    <property name="scaledContents">
     <bool>true</bool>
    </property>
   </widget>
   <widget class="QLineEdit" name="passLineEdit">
    <property name="geometry">
     <rect>
      <x>42</x>
      <y>41</y>
      <width>190</width>
      <height>29</height>
     </rect>
    </property>
    <property name="focusPolicy">
     <enum>Qt::StrongFocus</enum>
    </property>
    <property name="toolTip">
     <string>Contraseña.</string>
    </property>
    <property name="echoMode">
     <enum>QLineEdit::Password</enum>
    </property>
   </widget>
   <widget class="QCheckBox" name="savePassCheckBox">
    <property name="geometry">
     <rect>
      <x>42</x>
      <y>70</y>
      <width>188</width>
      <height>22</height>
     </rect>
    </property>
    <property name="toolTip">
     <string>Recordar usuario y contraseña</string>
    </property>
    <property name="text">
     <string>Recordar                               </string>
    </property>
   </widget>
  </widget>
  <widget class="QPushButton" name="cancellPushButton">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>270</y>
     <width>111</width>
     <height>27</height>
    </rect>
   </property>
   <property name="text">
    <string>&amp;Desconectar</string>
   </property>
   <property name="icon">
    <iconset resource="systray.qrc">
     <normaloff>:/images/disconnect.png</normaloff>:/images/disconnect.png</iconset>
   </property>
  </widget>
  <widget class="QPushButton" name="acceptPushButton">
   <property name="geometry">
    <rect>
     <x>130</x>
     <y>270</y>
     <width>111</width>
     <height>27</height>
    </rect>
   </property>
   <property name="text">
    <string>&amp;Conectar</string>
   </property>
   <property name="icon">
    <iconset resource="systray.qrc">
     <normaloff>:/images/connect.png</normaloff>:/images/connect.png</iconset>
   </property>
   <property name="default">
    <bool>true</bool>
   </property>
  </widget>
 </widget>
 <tabstops>
  <tabstop>userLineEdit</tabstop>
  <tabstop>passLineEdit</tabstop>
  <tabstop>savePassCheckBox</tabstop>
  <tabstop>domainLineEdit</tabstop>
  <tabstop>proxyLineEdit</tabstop>
  <tabstop>proxyPortSpinBox</tabstop>
  <tabstop>portSpinBox</tabstop>
  <tabstop>acceptPushButton</tabstop>
  <tabstop>cancellPushButton</tabstop>
  <tabstop>tabWidget</tabstop>
  <tabstop>spin1</tabstop>
 </tabstops>
 <resources>
  <include location="systray.qrc"/>
 </resources>
 <connections/>
</ui>
